### PR TITLE
Modal and forms are working.

### PR DESCRIPTION
**Pull request by sunkim0330**
I started working on styles yesterday, but didn't go too far with it.
My modal and forms in Modal are both working, and Modal is the only one that I styled a little bit so far.

@AlexArms @kate-darby @chrimack I know it's Sunday so there is no rush! I'm sorry I didn't do PR earlier 😖  Have a great day!

### CODE REVIEW

File: client/src/Q&As/ModalForm.jsx [hunk 1]
```diff
@@ -17,6 +19,28 @@ const ModalForm = ({origin, title, placeholder}) => {
...
+  const formValidation = () => {
+    let newErrors = {};
+    if (formData.body === '') {
+      newErrors.body = 'Please fill out this part';
+    }
+    if (formData.answerer_name === '') {
+      newErrors.answerer_name = 'Please fill out your username';
+    }
+    // let regEmail = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i;
+    let regEmail = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```
Comment (chrimack) [inline client/src/Q&As/ModalForm.jsx:32]: Whoa

File: client/src/Q&As/ModalForm.jsx [hunk 2]
```diff
@@ -17,6 +19,28 @@ const ModalForm = ({origin, title, placeholder}) => {
...
+    if (!regEmail.test(formData.email)) {
+      newErrors.email = 'Please enter a valid email address';
+    }
+    if (formData.email === '') {
+      newErrors.email = 'Mandatory Field';
+    }
+
+    setErrors(newErrors);
```
Comment (chrimack) [inline client/src/Q&As/ModalForm.jsx:40]: I like the way you use a state object for the errors

File: client/src/Q&As/Modal.jsx [hunk 3]
```diff
@@ -37,24 +31,31 @@ const Modal = ({origin, title, subTitle, questionBody, onClose, show}) => {
     }
   };
 
+  useEffect(() => {
+    if (isItAnswer) {
```
Comment (chrimack) [inline client/src/Q&As/Modal.jsx:23]: Good idea to set the modal like this to cut down repeptive code

**Review by chrimack — COMMENTED** [review]
This looks good to go

File: client/src/Q&As/Modal.jsx (hunk 3 above)
Comment (sunkim0330) [reply client/src/Q&As/Modal.jsx:23]: 😀

**Pull request merged**